docs/reference/commandline/run.md

```diff
@@ -17,6 +17,7 @@ parent = "smn_cli"
       -a, --attach=[]               Attach to STDIN, STDOUT or STDERR
       --add-host=[]                 Add a custom host-to-IP mapping (host:ip)
       --blkio-weight=0              Block IO weight (relative weight)
+      --blkio-weight-device="" 	    Block IO weight (relative weight for device)
```

Comment:
It's *after* `--blkio-weight-device=""`; see 
<img width="639" alt="screen shot 2015-10-13 at 17 50 31" src="https://cloud.githubusercontent.com/assets/1804568/10472225/fb500006-71d2-11e5-88ae-1cd78a2357ac.png">